### PR TITLE
Frontend/edit dest fixes

### PR DESCRIPTION
**Pull request by RitikaHanish**
## Type of Change
Frontend/edit dest fixes
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description
Frontend/edit dest fixes

### CODE REVIEW

**Review by MercyQueen — APPROVED** [review]

**Review by PraveenGongada — APPROVED** [review]

File: Frontend/ui-customer/src/Services/API.purs [hunk 1]
```diff
@@ -3815,12 +3815,18 @@ newtype BookingUpdateRequestDetails = BookingUpdateRequestDetails {
   estimatedFare :: Maybe Number,
   id :: String,
   oldEstimatedDistance :: Maybe Int,
+  errorObj :: Maybe ErrorObj,
```
Comment (ashishsingh101) [inline Frontend/ui-customer/src/Services/API.purs:3818]: why error obj is added inside success response ? this looks like temporary fix, can you please explain this use case

Comment (witcher-shailesh) [reply Frontend/ui-customer/src/Services/API.purs:3818]: I think it's not actual error but when a driver rejects the request to update the destination location

File: Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/callbacks/ShowNotificationCallBack.java [hunk 2]
```diff
@@ -6,4 +6,5 @@
 
 public interface ShowNotificationCallBack {
     void showInAppNotification(JSONObject jsonObject, Context context);
+    void hideInAppNotification(String channelId);
```
Comment (ashishsingh101) [inline Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/callbacks/ShowNotificationCallBack.java:9]: hide notification channelId is same as that of showNotification ?

Comment (RitikaHanish) [reply Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/callbacks/ShowNotificationCallBack.java:9]: yes @ashishsingh101

**Pull request merged**